migration/migrator/data/course_tables.sql

```diff
@@ -168,6 +168,30 @@ CREATE SEQUENCE public.categories_list_category_id_seq
 ALTER SEQUENCE public.categories_list_category_id_seq OWNED BY public.categories_list.category_id;
 
 
+--
+-- Name: course_materials; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.course_materials (
+    path text NOT NULL,
```

Comment:
A though occurs while looking through this PR, would we benefit here from having a regular serial primary key `id` here and make `path` a unique key on the table? That would allow not having to worry about dealing with serialization/unserialization errors on GET paths (e.g. like on deleting a course material).